module/Decision/src/Decision/Mapper/Member.php

```diff
@@ -60,6 +61,36 @@ public function searchByName($query, $maxResults = 32)
     }
 
     /**
+     * Find all members with a birthday in the next $days days.
+     *
+     * When $days equals 0 or isn't given, it will give all birthdays of today.
+     *
+     * @param int $days The number of days to look ahead.
+     *
+     * @return array Of members sorted by birthday
+     */
+    public function findBirthdayMembers($days)
+    {
+        // unfortunately, there is no support for functions like DAY() and MONTH()
+        // in doctrine2, thus we have to use the NativeSQL here
+        $builder = new ResultSetMappingBuilder($this->em);
```

Comment:
Native SQL is hier niet nodig. Je kunt hier een mapper functie van maken die verjaardagen uit een specifieke datum range ophaalt. Vervolgens kun je in de service eenvoudig een datum range berekenen. E.g. (new DateTime())->add(new DateInterval('P7D')). Zo is dit ook in de foto module opgelost https://github.com/GEWIS/gewisweb/blob/master/module/Photo/src/Photo/Mapper/Album.php#L171